packages/js/block-templates/src/hooks/use-layout-template/use-layout-template.ts

```diff
@@ -15,7 +15,7 @@ import { useEntityRecord } from '@wordpress/core-data';
 // eslint-disable-next-line @woocommerce/dependency-group
 import { dispatch, select } from '@wordpress/data';
 
-export const useLayoutTemplate = ( layoutTemplateId: string | undefined ) => {
+export const useLayoutTemplate = ( layoutTemplateId: string | null ) => {
```

Comment:
Why this change? `undefined` seems reasonable too.

Reply:
I prefer using `null` whenever I want to pass an explicit "falsy" value to something. Previously, this value was never actually `undefined`, but I started passing `null` before it is resolved in this PR